src/event.js

```diff
@@ -226,6 +226,63 @@ geo_event.unzoomselect = 'geo_unzoomselect';
 
 //////////////////////////////////////////////////////////////////////////////
 /**
+ * Triggered when an action is initiated with mouse down
+ *
+ * @property {object} state The action state
+ * @property {geo.mouseState} mouse The mouse state
+ * @property {object} event The triggering event
+ */
+//////////////////////////////////////////////////////////////////////////////
+geo_event.actiondown = 'geo_actiondown';
```

Comment:
We haven't been consistent: `geo_event.brushend = geo_brushend`, for instance.  I can change the actions to camel case or not as you prefer.  I'd be tempted to make the events consistent in casing, but that would be a breaking change for not much benefit.